DAO.sol

```diff
@@ -821,7 +822,10 @@ contract DAO is DAOInterface, Token, TokenCreation {
 
 
     function halveMinQuorum() returns (bool _success) {
-        if (lastTimeMinQuorumMet < (now - quorumHalvingPeriod)) {
+        // this can only be called after `quorumHalvingPeriod` has passed or at anytime
+        // by the curator with a delay of at least `minProposalDebatePeriod` between the calls
+        if ((lastTimeMinQuorumMet < (now - quorumHalvingPeriod) || msg.sender == curator)
+			&& lastTimeMinQuorumMet < (now - minProposalDebatePeriod)) {
```

Comment:
@CJentzsch My problem here is that there is no way to know if the Curator is malicious until all your ETH has been disappeared. The time between the requirement for a malicious action and all your ETH being gone is 15 seconds. 

Consider the following:
1. I create 1,000 proposals to send 1 ETH, I have 15% of DAO. 
2. I vote for my proposals no one else does. It will not meet the quorum. I will lose my deposit
3. At the last second the curator reduces teh quorum. I execute the proposals in the next block. All eth goes to me.